packages/dx-react-demos/src/theme-sources/bootstrap4/demo-source-data.json

```diff
@@ -0,0 +1,5 @@
+{
+  "additionalImports": "",
+  "openWrapperTag": "div className='card'",
```

Comment:
Why do you use different tags for `openWrapperTag` and `closeWrapperTag`? It may be confusing that the variable which is holding tag name also contains `className`.
Can we use reactstrap `Card` instead?